molgenis-data/src/main/java/org/molgenis/data/event/BootstrappingEvent.java

```diff
@@ -0,0 +1,16 @@
+package org.molgenis.data.event;
+
+public class BootstrappingEvent
```

Comment:
Maybe we can generify this a bit to support multiple bootstrapping 'states'? With an Enum for the states for example. `STARTED`, `POPULATING`, `FINISHED`.. something like that.